Gifski/MainWindowController.swift

```diff
@@ -296,25 +46,23 @@ final class MainWindowController: NSWindowController {
 		}
 	}
 
-	private func setupTimeRemainingLabel() {
-		guard let view = view else {
-			return
+	func convert(_ inputUrl: URL) {
+		if let dropVideo = window?.contentViewController as? DropVideoViewController {
+			dropVideo.convert(inputUrl)
+		} else if !(window?.contentViewController is ConversionViewController) {
+			let dropVideo = DropVideoViewController()
+			window?.contentViewController?.push(viewController: dropVideo) {
+				dropVideo.convert(inputUrl)
+			}
 		}
```

Comment:
Is there really no better way of doing this?

Reply:
There must be, we don't need the drop video controller here 😄 Will be fixed.